dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProjectServiceImpl.java

```diff
@@ -538,6 +549,18 @@ public Result queryProjectCreatedAndAuthorizedByUser(User loginUser) {
         return result;
     }
 
+    /**
+     * check whether have read permission
+     *
+     * @param user user
+     * @param project project
+     * @return true if the user have permission to see the project, otherwise return false
+     */
+    private boolean checkReadPermission(User user, Project project) {
```

Comment:
Please confirm whether it is referenced

Reply:
Sorry, this method was not referenced and was incorrectly merged while I was handling the conflict.